support/macros.c

```diff
@@ -9,6 +9,7 @@
 #include <dirent.h>
 #include <string.h>
 #include <glib.h>
+#include "mph.h" /* Don't remove! Works around issues with Android SDK unified headers */
```

Comment:
These comments should be worded consistently and also appear at the other places where you're moving this directive above the `#include "map.h"`.